.gitlab/llnl-ci-conda.yml

```diff
@@ -138,7 +138,7 @@ conda-tests:
     - export contact="arambula2@llnl.gov"
     - export imnumber="LLNL-MI-834241"
     - python3 ci/tests/test.py -s
-    - for TEST in parsl rp swift-t; do
+    - for TEST in flux parsl rp parsl-flux rp-flux maestro; do
```

Comment:
`maestro` is not in conda
(besides this, all looks good)

Reply:
Done